diff --git a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
index 97f698d40bbac04bb81c9f2abea94653eacf128b..b3f866de1aea850b2d218fa0c556f52b615b5fe2 100644
--- a/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
+++ b/chrome/browser/webauthn/chrome_authenticator_request_delegate.cc
@@ -248,6 +248,7 @@ ChromeWebAuthenticationDelegate::TouchIdAuthenticatorConfigForProfile(
         kWebAuthnTouchIdMetadataSecretPrefName,
         base::Base64Encode(base::as_bytes(base::make_span(metadata_secret))));
   }
+  BRAVE_WEBAUTHN_KEYCHAIN_ACCESS_GROUP
 
   return TouchIdAuthenticatorConfig{
       .keychain_access_group = kKeychainAccessGroup,
