diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 4bebe19d505b331e3cfbc4ea77723fcece227687..b2fff713a9143e88898dff839c8d66b51eff768c 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2480,6 +2480,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
