diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
index edf31ba76eb0e2949aaa916d2f83fb55a8dc7b03..c88c72b3f3a4774264c831b8765137f3cf2bdf37 100644
--- a/components/content_settings/core/common/content_settings.cc
+++ b/components/content_settings/core/common/content_settings.cc
@@ -97,6 +97,7 @@ constexpr HistogramValue kHistogramValue[] = {
     {ContentSettingsType::FILE_SYSTEM_ACCESS_CHOOSER_DATA, 76},
     {ContentSettingsType::FEDERATED_IDENTITY_SHARING, 77},
     {ContentSettingsType::FEDERATED_IDENTITY_REQUEST, 78},
+    BRAVE_HISTOGRAM_VALUE_LIST
 };
 
 }  // namespace
@@ -183,6 +184,7 @@ bool RendererContentSettingRules::IsRendererContentSetting(
          content_type == ContentSettingsType::JAVASCRIPT ||
          content_type == ContentSettingsType::CLIENT_HINTS ||
          content_type == ContentSettingsType::POPUPS ||
+         BRAVE_IS_RENDERER_CONTENT_SETTING
          content_type == ContentSettingsType::MIXEDSCRIPT;
 }
 
