diff --git a/extensions/browser/user_script_manager.h b/extensions/browser/user_script_manager.h
index a28d077ef73fe65cc646c60ee1282b1ee3a3fefa..f5c770af77528b1ee0d19fb2e3ea12ce945d2cc7 100644
--- a/extensions/browser/user_script_manager.h
+++ b/extensions/browser/user_script_manager.h
@@ -24,7 +24,7 @@
 namespace content {
 class BrowserContext;
 }
-
+class BraveWalletService;
 namespace extensions {
 class UserScriptLoader;
 
@@ -48,6 +48,7 @@ class UserScriptManager : public ExtensionRegistryObserver {
   WebUIUserScriptLoader* GetUserScriptLoaderForWebUI(const GURL& url);
 
  private:
+  friend class ::BraveWalletService;
   // ExtensionRegistryObserver implementation.
   void OnExtensionLoaded(content::BrowserContext* browser_context,
                          const Extension* extension) override;
