diff --git a/services/network/restricted_cookie_manager.cc b/services/network/restricted_cookie_manager.cc
index a921e988616f67f12a79faeec6e467a6cb7cb40a..3c804575c725266045e39d790778246c5a52d79d 100644
--- a/services/network/restricted_cookie_manager.cc
+++ b/services/network/restricted_cookie_manager.cc
@@ -301,6 +301,7 @@ void RestrictedCookieManager::GetAllForUrl(
   //                                 removing deprecation warnings.
   net_options.set_return_excluded_cookies();
 
+  BRAVE_GETALLFORURL
   cookie_store_->GetCookieListWithOptionsAsync(
       url, net_options,
       base::BindOnce(&RestrictedCookieManager::CookieListToGetAllForUrlCallback,
@@ -452,6 +453,7 @@ void RestrictedCookieManager::SetCanonicalCookie(
       role_, url, site_for_cookies, isolation_info_, cookie_settings(),
       cookie_store_->cookie_access_delegate());
 
+  BRAVE_SETCANONICALCOOKIE
   cookie_store_->SetCanonicalCookieAsync(
       std::move(sanitized_cookie), origin_url, options,
       base::BindOnce(&RestrictedCookieManager::SetCanonicalCookieResult,
